dev/tasks/tasks.yml

```diff
@@ -1549,9 +1549,7 @@ tasks:
       image: conda-python-hdfs
 {% endfor %}
 
-{% for python_version, spark_version, test_pyarrow_only, numpy_version, jdk_version in [("3.8", "v3.5.0", "false", "latest", "8"),
-                                                                                        ("3.10", "v3.5.0", "false", "1.23", "8"),
-                                                                                        ("3.11", "master", "false", "latest", "17")] %}
+{% for python_version, spark_version, test_pyarrow_only, numpy_version, jdk_version in [("3.11", "master", "false", "latest", "17")] %}
```

Comment:
Can we also delete the maven plugin artifacts from this file?

```
      - arrow-maven-plugins-{no_rc_snapshot_version}-cyclonedx.json
      - arrow-maven-plugins-{no_rc_snapshot_version}-cyclonedx.xml
      - arrow-maven-plugins-{no_rc_snapshot_version}.pom
      - module-info-compiler-maven-plugin-{no_rc_snapshot_version}-cyclonedx.json
      - module-info-compiler-maven-plugin-{no_rc_snapshot_version}-cyclonedx.xml
      - module-info-compiler-maven-plugin-{no_rc_snapshot_version}-javadoc.jar
      - module-info-compiler-maven-plugin-{no_rc_snapshot_version}-sources.jar
      - module-info-compiler-maven-plugin-{no_rc_snapshot_version}.jar
      - module-info-compiler-maven-plugin-{no_rc_snapshot_version}.pom
 ```

Reply:
I'm surprised it was not already included but yes, those lines should be removed as well